src/dbnode/storage/bootstrap_test.go

```diff
@@ -64,7 +64,7 @@ func TestDatabaseBootstrapWithBootstrapError(t *testing.T) {
 	bsm.sleepFn = func(time.Duration) {}
 
 	gomock.InOrder(
-		ns.EXPECT().GetOwnedShards().Return([]databaseShard{}),
+		ns.EXPECT().PrepareBootstrap().Return([]databaseShard{}, nil),
```

Comment:
Can't comment on the line unfortunately, but maybe update the Controller to be an `xtest.NewController(t)`?